packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -7,26 +7,32 @@ import {
   serverSatisfies,
   skipForWeb,
   TEST_MULTIPLE_CONNECTIONS,
-  connectionNameFromString,
-  DEFAULT_CONNECTION_STRING,
 } from '../helpers/compass';
 import type { Compass } from '../helpers/compass';
 import * as Selectors from '../helpers/selectors';
 import { getFirstListDocument } from '../helpers/read-first-document-content';
 import { MongoClient } from 'mongodb';
 
 import delay from '../helpers/delay';
+import type { ConnectFormState } from '../helpers/connect-form-state';
 
 const CONNECTION_HOSTS = '127.0.0.1:27091';
 const CONNECTION_STRING = `mongodb://${CONNECTION_HOSTS}/`;
 
-async function refresh(browser: CompassBrowser) {
+async function refresh(browser: CompassBrowser, connectionName: string) {
```

Comment:
This file is where most of the action happens, but github collapses it by default.